.github/workflows/flutter-ci.yml

```diff
@@ -0,0 +1,104 @@
+name: Flutter CI
+
+# This workflow is triggered on pushes to the repository.
+
+on:
+  push:
+    branches: [dev]
+  pull_request: 
+    branches: [dev]
```

Comment:
Can we add in staging and master to this? I'm also thinking we wont want this to run on pr once we merge it in
```suggestion
  push:
    branches: [dev, staging, master]
```

Reply:
Do staging builds need to be built differently?

Reply:
They used to be but for now let's keep it simple

Reply:
I think its probably better if we just keep them the same